extension/src/test/fixtures/expShow/deeplyNested/maxHeight.ts

```diff
@@ -352,20 +352,21 @@ export const deeplyNestedColumnsWithHeightOf2: Column[] = [
 export const deeplyNestedColumnsWithHeightOf1: Column[] = [
```

Comment:
[F] This fixture was actually broken. The paths were wrong and one column was even missing. Bug demonstrated here:

https://github.com/iterative/vscode-dvc/assets/37993418/ad766e37-9475-44a7-af41-49d375076621

and fixed:

https://github.com/iterative/vscode-dvc/assets/37993418/b1a5ac9b-4990-482a-a768-093eda5bab06